pyproject.toml

```
@@ -0,0 +1,26 @@
+[tool.poetry]
+name = "django-ibmi"
+version = "0.1.0-dev"
+description = "'Db2 for i support for Django framework'"
```

```suggestion
description = "Db2 for i adapter for Django"
```

Is there a need for the inner quotes?